Sources/Wrap.swift

```diff
@@ -120,13 +120,13 @@ public protocol WrapCustomizable {
      *  Returning nil from this method will be treated as an error, and cause
      *  a `WrapError.WrappingFailedForObject()` error to be thrown.
      */
-    func wrap() -> AnyObject?
+    func wrap(context: Any?) -> AnyObject?
```

Comment:
Agree to that and was thinking about it too. 
But with Swift3 [all labels will show same behaviour](https://github.com/apple/swift-evolution/blob/master/proposals/0046-first-label.md), means the first parameter name will also be its external name and wouldn't need an explicit parameter name anymore.

We could add explicit parameter naming now to have Swift3 style naming under Swift2 and remove it later when swift3 is released.  But if we do so shouldn't it be consistent throughout the lib and should use swift3 naming for all functions? 
`Wrapper(_:context:)` -> `Wrapper(dateFormatter:context:)`,
`keyForWrappingPropertyNamed(_:context:)` -> `keyForWrapping(propertyName:context:)`,
...

Or should Swift3 Naming be introduced in one commit when swift3 is released and this lib supports it officially?